src/main/java/org/openstreetmap/atlas/geography/Polygon.java

```diff
@@ -55,6 +55,7 @@
             Location.forString("37.39018121506223,-122.03110367059708"),
             Location.forString("37.39021104996917,-122.0310714840889"),
             Location.forString("37.390234491673446,-122.03111171722412"));
+    public static final Polygon CENTER = new Polygon(Location.forString("0,0"));
```

Comment:
Good call. I would note that sonar complains about this and the other example `Polygon` declarations, since they expose a static mutable object. Since we only use them for testing purposes I am tempted to `// NOSONAR` all the warnings. Perhaps instead we can ignore them in the sonar UI?

Reply:
The other solution, which would appease sonar, would could introduce a breaking change like:
```
- public static final Polygon CENTER = new Polygon(Location.CENTER);
+ public static Polygon getCenter()
+ {
+     return new Polygon(Location.CENTER)
+ }
```
This would require changing unit/int tests. Not sure how widespread usage of `CENTER` would be in downstream repositories. I would guess relatively minimal impact.

Perhaps this is worth doing in a separate sonar cleanup PR?